src/ui/form/Button.tsx

```diff
@@ -1,31 +1,74 @@
 import { FC } from 'react';
 import styled from '@emotion/styled';
 
-const StyledButton = styled.button`
+const DefaultButton = styled.button`
   border: 1px var(--text) solid;
-  height: 30px;
   border-radius: 5px;
-  background: var(--input-background);
-  padding: 3px;
   &:disabled {
     opacity: 0.2;
   }
 `;
 
+const SmallButton = styled(DefaultButton)`
+  padding: 5px;
+`;
+
+const SalmonButton = styled(DefaultButton)`
+  background: var(--text);
+  color: var(--background);
+  padding: 10px;
+`;
+
+const WineButton = styled(DefaultButton)`
+  background: var(--input-background);
+  padding: 10px;
+`;
+const OchreButton = styled(DefaultButton)`
```

Comment:
we should only have one button (in this file at least). anything you want to change should be passed as props. 

you could define other Buttons (in other files) using this base Button component if you want.

but most of these could be handled with props (see the Flex component for the pattern of passing props to styling)